presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java

```diff
@@ -50,6 +54,10 @@
     private int maxConnectionIdleTime;
     private boolean socketKeepAlive;
     private boolean tlsEnabled;
+    private File keystorePath;
```

Comment:
This should be captured as separate config class and it should be only binded when `tlsEnabled == true`